solr/CHANGES.txt

```diff
@@ -128,6 +128,8 @@ when told to. The admin UI now tells it to. (Nazerke Seidan, David Smiley)
 
 * SOLR-15340: Rename shardsWhitelist and extract AllowListUrlChecker to use it more broadly. (Bruno Roustant)
 
+* SOLR-15315: LTR latency optimisation by changing recursion to loop in tree traversal. (Suryakant Pandey)
```

Comment:
Add Christine in here as well, please :) she helped with some code and review effort, it's good to give merit

Reply:
Alessandro too I think :)

Added https://github.com/apache/solr/pull/58/commits/d522b04186783a0d451d5c3f70e2fc9cef0f4e33 edit (details in the commit message) and will merge shortly.